server/web/git.go

```diff
@@ -457,6 +478,7 @@ func getInfoRefs(w http.ResponseWriter, r *http.Request) {
 		}
 
 		user := proto.UserFromContext(ctx)
+		cmd.Env = append(cmd.Env, cfg.Environ()...)
```

Comment:
I think here you probably want `cmd.Environ()` instead of `cmd.Env` (which will be empty)